<a href="https://colab.research.google.com/github/jsleocadio/-IMD0184---APRESENTACAO-E-ANALISE-ESTATISTICA-DE-DADOS/blob/main/Guia_2___Jefferson_Leocadio.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

### Banco de dados – Qual base de dados você escolheu para tratar o seu problema?
A partir dos dados é possível obter o relacionamento entre as variáveis e buscar respostas para validar ou refutar as suas hipóteses.
Deve-se ter em mente que, quanto mais dados se tem sobre o tema, mais informação é possível gerar a respeito. Preencha as
informações indicadas no espaço abaixo:
* Local de onde a base foi extraída ou obtida (site, repositório, etc.);
* Número de observações ou unidades amostrais da base;
* Número de variáveis ou atributos da base;
* Apresenta dicionário ou explicação sobre o que são as variáveis?

In [1]:
import pandas as pd
import numpy as np
from scipy.fft import fft
import matplotlib.pyplot as plt
from matplotlib.axis import XTick
import matplotlib.dates

In [2]:
filename = 'https://raw.githubusercontent.com/jsleocadio/Signal-Digital-Processing/main/Project_01/2022-09-28_Covid-Bra.csv'
df = pd.read_csv(filename)
df.head()

iso_code      continent location        date  total_cases  new_cases  \
0      BRA  South America   Brazil  2020-02-26            1        1.0   
1      BRA  South America   Brazil  2020-02-27            1        0.0   
2      BRA  South America   Brazil  2020-02-28            1        0.0   
3      BRA  South America   Brazil  2020-02-29            2        1.0   
4      BRA  South America   Brazil  2020-03-01            2        0.0   

   new_cases_smoothed  total_deaths  new_deaths  new_deaths_smoothed  ...  \
0                 NaN           NaN         NaN                  NaN  ...   
1                 NaN           NaN         NaN                  NaN  ...   
2                 NaN           NaN         NaN                  NaN  ...   
3                 NaN           NaN         NaN                  NaN  ...   
4                 NaN           NaN         NaN                  NaN  ...   

   female_smokers  male_smokers  handwashing_facilities  \
0            10.1          17.9                     NaN   
1            10.1          17.9                     NaN   
2            10.1          17.9                     NaN   
3            10.1          17.9                     NaN   
4            10.1          17.9                     NaN   

   hospital_beds_per_thousand  life_expectancy  human_development_index  \
0                         2.2            75.88                    0.765   
1                         2.2            75.88                    0.765   
2                         2.2            75.88                    0.765   
3                         2.2            75.88                    0.765   
4                         2.2            75.88                    0.765   

   excess_mortality_cumulative_absolute  excess_mortality_cumulative  \
0                                   NaN                          NaN   
1                                   NaN                          NaN   
2                                   NaN                          NaN   
3                                7049.6                         3.31   
4                                   NaN                          NaN   

   excess_mortality  excess_mortality_cumulative_per_million  
0               NaN                                      NaN  
1               NaN                                      NaN  
2               NaN                                      NaN  
3              5.25                                32.891915  
4               NaN                                      NaN  

[5 rows x 67 columns]

###Variável resposta – Qual variável você acredita que pode te ajudar a responder suas perguntas?
Qual a variável, dentre todas as presentes na sua base de dados, foi a escolhida como resposta? Por quê?
* Justifique a sua escolha relacionando com o problema e as hipóteses que guiam o seu trabalho;
* Qual o tipo da sua variável? Quantitativa ou Qualitativa?
* Qual o domínio da sua variável ou quais as categorias ela possui?

New Deaths, pois precisamos verificar se houve redução de mortes na medida em que houve aumento na vacinação da população

###Resumo da sua variável resposta – Como é que se comporta a sua variável?
Nesta etapa deve-se buscar compreender como é essa variável. Qual a média? Qual o desvio padrão? Qual a proporção das categorias dessa
variável? Tem outliers? Tem valores faltantes? Quais os mínimos e máximo? Faça gráficos ou tabelas para auxiliar o processo de conhecimento da
sua variável resposta.

In [4]:
Ind = ~np.isnan(df.new_deaths)
df = df[Ind]

In [5]:
df['new_deaths'].describe()

count     924.000000
mean      742.435065
std       742.791153
min         0.000000
25%       201.750000
50%       542.000000
75%      1063.000000
max      4148.000000
Name: new_deaths, dtype: float64

###Traduzindo – O que tudo isso quis dizer?
O objetivo aqui será a síntese do que se obteve com a análise da variável resposta de forma textual, relacionando com o tema, problema e hipóteses
do seu trabalho.


Podemos observar que da amostra de novas mortes por covid temos 924 valores, sendo seu menor valor nenhuma morte e o pico sendo 4148 mortes em um dia. A média de mortes no período foi de 742 mortes no período avaliado e o desvio padrão de 743 mortes.

###Variáveis Explicativas – Que outras variáveis você pode dispor para te ajudar na análise?
Nessa etapa você vai identificar as variáveis ou atributos da sua base de dados que terão a característica de auxiliar a compreender a
sua variável resposta. A essas variáveis é dado o nome de “Explicativas ou Variáveis Independentes”. Escolha um número de
variáveis e dê informações sobre elas:
* Essas variáveis são qualitativas ou quantitativas? Qual o domínio dessas variáveis? Quais categorias elas têm
* Justifique a sua escolha relacionando com o problema, as hipóteses que guiam o seu trabalho e a variável resposta definida;

* Vacinação total
* Taxa de Rigidez